### PR TITLE
Pass kwargs into dfs call in get_valid_primitives to allow other dfs parameters to be passed in

### LINKED ISSUE #2158: Allow other dfs parameters to be used in get_valid_primitives
- As a user, I wish I could use Featuretools' `get_valid_primitives` with any additional dfs keyword arguments that I may like to have applied when generating valid primitives, such as `ignore_columns` or `ignore_dataframes`

### PR DESCRIPTION
**Pull request by tamargrey**
closes #2158

### CODE REVIEW

**Review by thehomebrewnerd — APPROVED** [review]
Looks good - just one small spelling error.

File: featuretools/synthesis/get_valid_primitives.py [hunk 1]
```diff
@@ -9,7 +9,11 @@
 
 
 def get_valid_primitives(
-    entityset, target_dataframe_name, max_depth=2, selected_primitives=None
+    entityset,
+    target_dataframe_name,
+    max_depth=2,
+    selected_primitives=None,
+    **dfs_kwargs,
```
Comment (rwedge) [inline featuretools/synthesis/get_valid_primitives.py:16]: Would `**kwargs` be more recognizable?  We could rely on the docstring to specify valid keywords

Comment (tamargrey) [reply featuretools/synthesis/get_valid_primitives.py:16]: For sure. I wasn't sure if it was good to specify in the art name where the keyword args would get used, but I'm happy to make it more recognizable.

Comment (rwedge) [reply featuretools/synthesis/get_valid_primitives.py:16]: After more time to consider I like the specificity of `**dfs_kwargs`

Comment (tamargrey) [reply featuretools/synthesis/get_valid_primitives.py:16]: Sounds good. If you saw the push with this change before I saw this comment---no you didnt!

File: featuretools/synthesis/get_valid_primitives.py [hunk 2]
```diff
@@ -30,6 +34,9 @@ def get_valid_primitives(
         selected_primitives(list[str or AggregationPrimitive/TransformPrimitive], optional):
             list of primitives to consider when looking for valid primitives.
             If None, all primitives will be considered
+        dfs_kwargs (keywords): Additional keyword arguments to pass as keyworg arguments to
```
[the remark this answers is not in the document]
Comment (tamargrey) [reply featuretools/synthesis/get_valid_primitives.py]: Changed!

**Pull request merged**